man-roxygen/table-relevance.R

```diff
@@ -9,6 +9,6 @@
 #'
 #' \deqn{recall = A/(A+C)}
 #' \deqn{precision = A/(A+B)}
-#' \deqn{F_meas_\beta = (1+\beta^2) * precision * recall/((\beta^2 * precision)+recall)}
+#' \deqn{F_{meas}_\beta = (1+\beta^2) * precision * recall/((\beta^2 * precision)+recall)}
```

Comment:
```suggestion
#' \deqn{F_{meas} = (1+\beta^2) * precision * recall/((\beta^2 * precision)+recall)}
```
Based on chatting with @DavisVaughan